pkg/service/schema.go

```diff
@@ -100,6 +100,19 @@ func (a *ArrayParameterSchema) MarshalJSON() ([]byte, error) {
 	})
 }
 
+// NumericParameterSchema represents a numeric type, either
+// integers or floating point numbers, that can have an upper or lower bound.
+type NumericParameterSchema struct {
+	Type             string      `json:"type"`
+	Description      string      `json:"description,omitempty"`
+	Required         bool        `json:"-"`
+	Default          interface{} `json:"default,omitempty"`
+	Minimum          interface{} `json:"minimum,omitempty"`
+	ExclusiveMinimum bool        `json:"exclusiveMinimum,omitempty"`
+	Maximum          interface{} `json:"maximum,omitempty"`
```

Comment:
These "Exlusive" properties aren't being used at the moment but are in the spec, so I included them.